tests/unit/test_graphstates.py

```diff
@@ -111,8 +111,22 @@ def test_SCZ_mat(self, random_graph):
             assert np.array_equal(mat[N:, :N], random_graph[1])
             assert np.array_equal(mat[N:, N:], np.identity(N))
 
-    # def test_SCZ_apply(self):
-    # pass
+    @pytest.mark.parametrize("one_shot", [True, False])
+    def test_SCZ_apply(self, random_graph, one_shot):
+        """test SCZ application with `one_shot` parameter"""
+        delta = rng().random()
+        model_init = noise_model(delta, "initial")
+        model_fin = noise_model(delta, "final")
+
+        n = len(random_graph[0])
+        G = CVLayer(random_graph[0])
+        G.apply_noise(model_init)
+        init_noise_all_GKP = np.full(2 * n, (delta / 2) ** 0.5, dtype=np.float32)
+        assert np.array_equal(G._init_noise, init_noise_all_GKP)
```

Comment:
I'm not sure this is necessary, since we are testing SCZ application here and there is no reference to SCZ.

Reply:
@sduquemesa do you agree? If this is here for the test coverage score, should we put it in a separate test?

Reply:
I think this test is not good and I wrote it for coverage. Let me come up with a better one.